projects/plugins/jetpack/extensions/shared/external-media/sources/with-media.js

```diff
@@ -169,6 +169,11 @@ export default function withMedia( mediaSource = MediaSource.Unknown ) {
 					parse: window.wpcomFetch === undefined,
 				} )
 					.then( result => {
+						// If we don't have media available, we should show an error instead of crashing the editor.
+						if ( result.media === undefined ) {
```

Comment:
```suggestion
						if ( ! Array.isArray( result.media ) ) {
```
We expect it to be an array, right? This is minor, but it seems like we might as well validate that it has the type we want instead of validating that it's not `undefined`

Reply:
I'm not sure if checking for Array here would trigger unknown side effects. What is sure is that when it's undefined the editor crashes.